AppController/djinn.rb

```diff
@@ -5359,7 +5461,8 @@ def get_scaling_info_for_app(app_name, update_dashboard=true)
     # Let's make sure we have the minimum number of AppServers running.
     Djinn.log_debug("Evaluating app #{app_name} for scaling.")
     if (@app_info_map[app_name]['appengine'].nil? ||
```

Comment:
Can we assign a variable to `@app_info_map[app_name]['appengine']` so that the length of this if decreases?